cocotb/clock.py

```diff
@@ -62,9 +64,12 @@ class Clock(BaseClock):
 
     Args:
         signal (pin): The clock pin/signal to be driven.
-        period (int): The clock period. Should be an even number.
+        period (int): The clock period. Must convert to an even number of
+            timesteps.
         units (str, optional): One of (None,'fs','ps','ns','us','ms','sec').
-            Defaults to ``None``.
+            When no units are given (``None``) the timestep is determined by
+            the simulator. E.g. Cadence simulators have a default timestep of
```

Comment:
I would not single out any specific vendor here, but just say something like "the timestep depends on the simulator". (Personally, I always specify ``units`` because of that.)
(Related: #115).

Reply:
Oh it's just there as example. Would you prefer that we really not have it?